_posts/2016-02-16-ten-steps-to-evaluate-a-rails-project.markdown

```diff
@@ -0,0 +1,136 @@
+
+In the best case scenario, you will see something like this:
+
+```bash
+etagwerker:ombushop/ (master) $ bundle-audit
+No vulnerabilities found
+```
+
+## 7. Setup Code Climate
+
+This is the quick and (**paid**) step to check code quality. At [Ombu Labs](http://www.ombulabs.com) we have a business account that we use for our products and our client’s projects.
+
+[Code Climate](https://codeclimate.com) is a paid service that does automated code review for test coverage, complexity, duplication, security, and style.
+
+You can quickly find **code hotspots**, potential issues, potential bug risks, and even coverage stats.
+
+What is the GPA of the project? 4.0? 1.5? This will give you an overall idea of the complexity within the project. 4.0 is the best you can get!
+
+A free alternative would be to run some (or all) of these tools on the project:
+
+* [flog](https://rubygems.org/gems/flog)
+* [flay](https://rubygems.org/gems/flay)
+* [metric_fu](https://rubygems.org/gems/metric_fu)
+* [simplecov](https://rubygems.org/gems/simplecov)
+
+## 8. Check code coverage
+
+As much as I like [Code Climate](https://codeclimate.com), [simplecov](https://rubygems.org/gems/simplecov) is an even faster way to generate a quick report about code coverage.
+
+This tool will tell you the coverage percentage for each application layer. How much percentage is covered in the models' tests? How much for the controllers' tests?
+
+Which are the least covered files? How many relevant lines are in each file? How many of them are covered by a test?
+
+The overall test coverage percentage will give you an idea about how much past developers cared about tests.
+
+If you are going to be changing the code, you need to make sure that this code is covered. If not, it will be a nightmare!
+
+You should complement this report with a code review of the tests, to make sure that the coverage is actually thorough.
+
+## 9. Research the development process
+
+Do the developers have a process to introduce changes to the project? Do they use code reviews before they merge anything to `master`? Do they have a [CI](https://en.wikipedia.org/wiki/Continuous_integration) service that runs with every pull request?
+
+What's in the roadmap for the next 3 months? Is this project on fire? Do they need a firefighter? Why is it on fire?
+
+## 10. Review performance stats
+
+Do they use [Skylight](https://www.skylight.io)? [NewRelic](http://newrelic.com)? What are the slowest and busiest requests? What are the main performance problems?
+
+Most projects won't have information from these (paid) services. So you will have to find out on your own. If they use [Google Analytics](https://www.google.com/analytics/), you might find some information about poor page performance.
+
+## Conclusion
+
+These are the steps that we currently follow before we take over a project. They are quite helpful and I recommend that you use them to objectively judge the quality of the project.
```

Comment:
Maybe you could change "to objectively judge the quality of the project." to "to judge the quality of the project as objectively as possible."?